core/templates/dev/head/domain/skill/ConceptCardBackendApiService.ts

```diff
@@ -79,18 +79,18 @@ oppia.factory('ConceptCardBackendApiService', [
        * backend, it will store it in the cache to avoid requests from the
        * backend in further function calls.
        */
-      loadConceptCard: function(skillId) {
+      loadConceptCards: function(skillIds) {
         return $q(function(resolve, reject) {
-          if (_isCached(skillId)) {
+          if (_isCached(skillIds)) {
             if (resolve) {
-              resolve(angular.copy(_conceptCardCache[skillId]));
+              resolve(angular.copy(_conceptCardCache[skillIds]));
             }
           } else {
-            _fetchConceptCard(skillId, function(conceptCard) {
+            _fetchConceptCards(skillIds, function(conceptCards) {
               // Save the fetched conceptCard to avoid future fetches.
-              _conceptCardCache[skillId] = conceptCard;
+              _conceptCardCache[skillIds] = conceptCards;
```

Comment:
Are you making the key a list?

Reply:
Yes. Is that okay?

Reply:
See my above comment about list of skills as a key. I didn't understand why its required since a concept card should be linked to one skill.

Reply:
Done.